easybuild/easyconfigs/p/PGPLOT/PGPLOT-5.2.2-GCC-11.3.0.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'CmdCp'
+
+name = 'PGPLOT'
+version = '5.2.2'
+
+homepage = 'https://sites.astro.caltech.edu/~tjp/pgplot/'
+description = """The PGPLOT Graphics Subroutine Library is a Fortran- or C-callable,
+device-independent graphics package for making simple scientific graphs. It is intended
+for making graphical images of publication quality with minimum effort on the part of
+the user. For most applications, the program can be device-independent, and the output
+can be directed to the appropriate device at run time."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
```

Comment:
I think this can be at `GCCcore` toolchain level, see https://github.com/easybuilders/easybuild/wiki/Review-process-for-contributions#toolchain

Only reasons to go for `GCC` over `GCCcore` is if it contains Fortran compiled code. From the `sanity_check_paths` I get the idea you are only compiling the C libraries here, in which case `GCCcore` would be enough.

Reply:
You're right. I switched to GCCcore.